doc/index.rst

```diff
@@ -31,6 +31,7 @@ Install
 
     .. tab-item:: other
 
+        .. rst-class:: section-toc
         .. toctree::
             :maxdepth: 2
```

Comment:
Note: Since two days ([sphinx 7.4 release](https://www.sphinx-doc.org/en/master/changes.html#release-7-4-0-released-jul-15-2024)) you could do

```suggestion
        .. toctree::
            :class: section-toc
            :maxdepth: 2
```
but likely, we don't want to bump the dependency that aggressivly (would force evey dev environment to update).